RecordHub.CatalogService/RecordHub.CatalogService.Api/GrpcServices/CatalogCheckerService.cs

```diff
@@ -1,30 +1,27 @@
 ﻿using Grpc.Core;
-using Microsoft.Extensions.Logging;
+using RecordHub.CatalogService.Api.Protos;
 using RecordHub.CatalogService.Application.Data;
-
-namespace RecordHub.CatalogService.Infrastructure.Services
+namespace RecordHub.CatalogService.Api.GrpcServices
 {
     public class CatalogCheckerService : CatalogChecker.CatalogCheckerBase
     {
         private readonly ILogger<CatalogCheckerService> _logger;
         private readonly IUnitOfWork _repository;
+
         public CatalogCheckerService(ILogger<CatalogCheckerService> logger, IUnitOfWork unitOfWork)
         {
             _logger = logger;
             _repository = unitOfWork;
         }
+
         public override async Task<ProductReply> CheckProductExisting(ProductRequest request, ServerCallContext context)
         {
             var record = await _repository.Records.GetByIdAsync(Guid.Parse(request.ProductId));
-            bool isExisting = true;
-            if (record != null)
-            {
-                return new ProductReply { IsExisting = isExisting, Name = record.Name, Price = record.Price };
-            }
+            bool isExisting = record != null;
 
-            isExisting = false;
+            _logger.LogInformation($"Grpc call existing of product {request.ProductId}, Existing - {isExisting}");
 
-            return new ProductReply { IsExisting = isExisting };
+            return new ProductReply { IsExisting = isExisting, Name = record?.Name ?? null, Price = record?.Price ?? 0 };
```

Comment:
Guess, it's better to assign props on new line for all of them.
Like: new ()
{
      Prop1 = prop1,
      Prop2 = prop2,
}

As here not so little args, even several operations. Moreover, I think, a good practice to create its instance before return for more readability.